types/ahoy.js/index.d.ts

```diff
@@ -135,17 +132,14 @@ declare namespace ahoy {
     /** Send a single Ahoy tracking event. */
     function track(name: string, properties?: EventProperties): boolean;
 
-    /** Track all views and clicks. */
-    function trackAll(): void;
-
     /** Track form control changes. */
```

Comment:
Could you mark this function as deprecated? See https://github.com/ankane/ahoy.js/blob/a644e83ee99fde60c24cf2110b6b3f05a43f279a/src/index.js#L457

```suggestion
    /**
     * Track form control changes.
     * @deprecated trackChanges is deprecated and will be removed in 0.5.0
     */
```